xbmc/interfaces/python/swig.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
```

Comment:
Either remove this again or remove the `#pragma once` that's already there below the license comment.